### ISSUE TITLE
About ST-Net

### ISSUE THREAD

**Issue by Syderny** [maxpmx/HisToGene#2]
Hello!

I find that When I try to realize ST-Net and calculate its ARI on this dataset, the result is different from what the paper describes, though I have applied the default hyperparameters mentioned in the ST-Net paper.

So could you offer your trained ST-Net, or detailed hyperparameters you used to train (such as backbone(DenseNet or ResNet), epochs, learning rate, optimizer and so on)?

Thank you!

**Comment by maxpmx** [comment]
Since the original implementation of ST-Net is does not work in our machine, we implemented the ST-Net using PyTorch. The hyperparameters we used are (backbone='resnet101', lr=1e-6, n_epochs=50, optimizer='Adam'). How did you implement the ST-Net and how different are your results?

**Comment by Syderny** [comment]
I implemented it with (backbone='resnet101', lr=1e-5, n_epochs=100, optimizer='Adam'), and the average ARI and correlation are kind of higher. Maybe you could send me one of the trained models to help me find out the reason? My email address is yinr5@mail2.sysu.edu.cn. 

Thank you!
